.github/codeql/codeql-configuration.yml

```diff
@@ -6,3 +6,29 @@ paths:
   - Gulpfile.mjs
 paths-ignore:
   - src/lib
+
+# These queries appear to time out after the module conversion.
+# https://github.com/github/codeql/issues/10937
```

Comment:
Do we have an issue to track re-enabling this in a few weeks?

Reply:
#51401